x/mongo/driver/topology/topology.go

```diff
@@ -673,3 +673,41 @@ func (t *Topology) String() string {
 	}
 	return fmt.Sprintf("Type: %s, Servers: [%s]", desc.Kind, serversStr)
 }
+
+// isTopologyConsistent returns false when we have a replica set that claims to
+// have no primary but there exists a primary with all other nodes as secondaries. This
+// specifically works around HELP-13825.
+func (t *Topology) IsConsistent() bool {
```

Comment:
we don't need this anymore since we don't use it

Reply:
Awesome! removed